PURL-TYPES.rst

```diff
@@ -280,6 +280,20 @@ hex
       pkg:hex/bar@1.2.3?repository_url=https://myrepo.example.com
 
 
+huggingface
+------
+``huggingface`` for Hugging Face ML models
+
+- The default repository is ``https://huggingface.co``.
+- The ``namespace`` is the model repository username or organization, if present. It is case insensitive and must be lowercased.
+- The ``name`` is the model repository name. It is case insensitive and must be lowercased.
```

Comment:
> It is case insensitive and must be lowercased.

That is incorrect from my testing, it's case sensitive and shouldn't be lowercased

Reply:
Agreed. The name is case sensitive on Hugging Face.

https://huggingface.co/CompVis/stable-diffusion-v1-4
https://huggingface.co/CompVis/stable-diffusion-V1-4

Reply:
Updated spec + tests